v4-client-js/src/clients/native.ts

```diff
@@ -1182,3 +1182,23 @@ export async function cctpWithdraw(squidPayload: string): Promise<String> {
     return wrappedError(error);
   }
 }
+
+export async function getWithdrawalCapacityByDenom(denom: string): Promise<string> {
+  try {
+    const validatorClient = globalThis.client.validatorClient;
+    const result = await validatorClient.get.getWithdrawalCapacityByDenom(denom);
+    return encodeJson(result, ByteArrayEncoding.BIGINT);
+  } catch (e) {
+    return wrappedError(e);
+  }
+}
+
+export async function getWithdrawalAndTransferGatingStatus(): Promise<string> {
+  try {
+    const validatorClient = globalThis.client.validatorClient;
+    const result = await validatorClient.get.GetWithdrawalAndTransferGatingStatus();
+    return encodeJson(result, ByteArrayEncoding.BIGINT);
```

Comment:
is BIGINT encoding necessary for withdrawalAndTransferGatingStatus? I believe this should be human readable

Reply:
looks to be encoded
![Screenshot 2024-03-28 at 11 19 32 AM](https://github.com/dydxprotocol/v4-clients/assets/149746839/64dea250-7656-4267-809a-aee2cabfe0a6)